src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -323,6 +325,25 @@ def load_command_table(self, _):
         g.command('delete', 'delete')
         g.custom_command('set-primary', 'failover_group_failover')
 
+    ###############################################
+    #             sql instance-pool               #
+    ###############################################
+
+    instance_pools_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#InstancePoolsOperations.{}',
+        client_factory=get_sql_instance_pools_operations)
+    with self.command_group('sql instance-pool', instance_pools_operations, client_factory=get_sql_instance_pools_operations) as g:
+        g.command('show', 'get',
+                  table_transformer=instance_pool_table_format)
+        g.custom_command('list', 'instance_pool_list',
+                         table_transformer=instance_pool_table_format)
+        g.command('update', 'update')
+        g.command('delete', 'delete', supports_no_wait=True)
+        g.custom_command(
+            'create',
+            'instance_pool_create',
+            supports_no_wait=True)
```

Comment:
because your commands support no_wait, can we add wait command here?

Reply:
Included.